controls/srg_gpos/SRG-OS-000423-GPOS-00187.yml

```diff
@@ -8,6 +8,8 @@ controls:
             - package_openssh-server_installed
             - service_sshd_enabled
             - ssh_client_rekey_limit
```

Comment:
Looking at the RHEL8 STIG `SRG-OS-000424-GPOS-00188` keeps to be applicable for `ssh_client_rekey_limit` as well.